Makefile

```diff
@@ -104,4 +104,11 @@ uninstall:
 build-client: check-client build-dfget build-dfdaemon
 .PHONY: build-client
 
-build: check-client build-dfget build-dfdaemon check-supernode build-supernode
\ No newline at end of file
+# Create for the tool of generate docs of the client.
```

Comment:
I am afraid building binaries is very essential for developers. For example, I code on my Mac and have no docker environment, but I still use the Makefile to build binaries to test. 
Thus it is not for doc generation. How about changing that?